### ISSUE TITLE
Is ActiveSupport::TestCase supported?

### ISSUE THREAD

**Issue by WillTaylor22** [justinweiss/resque_unit#45]
Running the broken code below, I get this error:
```
NoMethodError:         NoMethodError: undefined method `assert_queued' for #<ApplicationTest:0x007ff8c3049c00>
```

It looks like ActiveSupport::TestCase may not have the required ResqueUnit::Assertions
Fixed code:
```
class ApplicationTest < ActiveSupport::TestCase
  # ActiveSupport::TestCase.send(:include, ResqueUnit::Assertions)
  include ResqueUnit::Assertions
```

Broken Code:
```
class ApplicationTest < ActiveSupport::TestCase
  
  def setup
    @application = applications(:joes_application)
    Resque.reset!
  end

  test "should start campaign when created" do
    Application.create(email: "charlie@gmail.com")
    
    assert_queued(RegistrationEmails::LeadCreatedJob)
  end
```

**Comment by justinweiss** [comment]
Hm, I wonder if this happened after Rails changed the test case that they inherited from. 

I'm doing a pretty major refactoring of resque_unit right now, and fixed a bunch of these bugs in the process. If you want to try that version, you can -- the only major caveat so far is that I don't have the resque_scheduler assertions working quite yet.

**Comment by justinweiss** [comment]
(that would be the https://github.com/justinweiss/resque_unit/tree/with-test-redis branch)

**Comment by WillTaylor22** [comment]
Ah nice one!

I am using the Resque scheduler stuff at the moment - but I got it all 100% working last night, so no rush from me

**Comment by justinweiss** [comment]
Hey, I just released a beta that should fix the issue -- if you want, try `v1.0.0.beta.1`.

**Comment by WillTaylor22** [comment]
Thanks!
